Vegetarian-Food-Finder/Api/ApiManger.swift

```diff
@@ -2,8 +2,35 @@
 //  ApiManger.swift
 //  Vegetarian-Food-Finder
 //
-//  Created by Abdullahi Ahmed on 7/6/22.
+//  Created by Abdullahi Ahmed on 7/7/22.
 //
 
 import UIKit
 
+final class ApiManger {
+    
+    private class ApiConstant {
+        static let ApiUrl = "https://api.yelp.com/v3/businesses/search?location=SF&categories=vegetarian"
+        static let apiKey = "_fWbIvhY5n5FFDT2iiUnJ2BKwupqLzTbqd7kFco46l6cioKrn2-VnnIU_97bEEJG5Tp-XeHsdcSYWTXV0tO4vFTzq61DbeLrbcA1d0g6wjQ46AzI2gnV3iuCvtHFYnYx"
+    }
+    
+    func makeBusinessRequest() {
```

Comment:
naming: I know that Yelp endpoint contains `business` in the URL. But try to name this method closer to what it actually does. Something like: `searchBusinesses`. You're not returning a request - it's used internally but not returned in the response, so I wouldn't include it in the method name.